### PR TITLE
Jia/fix: hyperlinks text and double line title

### PR DESCRIPTION
**Pull request by jia-deriv**
Fix hyperlinks text
Screenshot 2023-11-09 at 5 09 35 PM
Fix extra padding on desktop size that makes the title inside the card drop to the second line.
Screenshot 2023-11-09 at 5 10 53 PM
image

### CODE REVIEW

**Review by prince-deriv — DISMISSED** [review]

**Review by meenakshi-deriv — APPROVED** [review]

**Review by meenakshi-deriv — DISMISSED** [review]

File: libs/blocks/src/lib/features/card-feature/index.tsx [hunk 1]
```diff
@@ -42,7 +42,7 @@ const Card = ({
     >
       <FluidContainer
         className={qtMerge(
-          'flex flex-col items-center',
+          'flex flex-col items-center lg:px-general-none',
```
Comment (mohsen-deriv) [inline libs/blocks/src/lib/features/card-feature/index.tsx:45]: this fix should be in deriv-com/quill-design#69

Comment (mohsen-deriv) [reply libs/blocks/src/lib/features/card-feature/index.tsx:45]: let's have a chat on this @jia-deriv @prince-deriv , thanks 🙏

**Review by mohsen-deriv — CHANGES_REQUESTED** [review]

File: libs/blocks/src/lib/features/card-feature/index.tsx (hunk 1 above)
Comment (mohsen-deriv) [reply libs/blocks/src/lib/features/card-feature/index.tsx:45]: @jia-deriv can you please check if this is ok now? 
@prince-deriv 🙏

Comment (jia-deriv) [reply libs/blocks/src/lib/features/card-feature/index.tsx]: Done with removing the class from the `FluidContainer`

**Review by mohsen-deriv — APPROVED** [review]

**Pull request merged**